runtime/lua/vscode/internal.lua

```diff
@@ -280,4 +280,10 @@ function M.wslpath(path)
   return vim.trim(ret)
 end
 
+-- Get the current Neovim CWD
+-- @return string
+function M.cwd()
+  return fn.getcwd()
```

Comment:
Looks like unnecessary indirection. Why can't callers use `fn.getcwd()` directly?

Reply:
It was just to make use of the `internal` module with `actions.lua` As @xiyaowong It's not really needed

Reply:
that kind of thing will get copied and perpetuated. better to avoid it